docs/workflow-controller-configmap.yaml

```diff
@@ -221,7 +221,7 @@ data:
     path: /metrics
     # Port is the port where metrics are emitted. Default is "9090"
     port: 8080
-    # MetricsTTL sets how often custom metrics are cleared from memory. Default is "0", metrics are never cleared
+    # MetricsTTL sets how often custom metrics are cleared from memory. Default is "0", metrics are never cleared. Histogram metrics are never cleared.
```

Comment:
Maybe not clear enough, but [TTL no longer works](https://github.com/argoproj/argo-workflows/blob/40786c23e9e8d69b5cd76aeb9fb530779ecaa53f/docs/upgrading.md?plain=1#L50) for histogram metrics.

Reply:
Oh, the existing comment confused me since it says "metrics are never cleared" but it means that's what `"0"` means.
That would probably be good to clarify that comment 😅
Also a link to OTel might be good for histogram metrics, and I'd probably new line the whole sentence since it's pretty long as-is